src/test/java/io/vertx/test/core/FileResolverTestBase.java

```diff
@@ -230,6 +230,36 @@ public void testSendFileFromClasspath() {
     await();
   }
 
+  @Test
+  public void testSendOpenRangeFileFromClasspath() {
```

Comment:
I think it would be more appropriate to have these tests in HttpTest with the rest of the send file tests.

We do have some send file tests in FileResolverTest but these are just to make sure the offsetting is working ok.